tests/e2e/specs/out-of-game/api-status.spec.js

```diff
@@ -0,0 +1,32 @@
+const packageVersion = require('../../../../package.json').version;
```

Comment:
This can be ES6 syntax as far as I know:
```
const { version } = require('../../../../package.json');
```
Make sure it goes under the import below-- imports first, then conts.